tests/sample/bindmonitor_mt_tailcall.c

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+// clang -O2 -Werror -c bindmonitor_mt_tailcall.c -o bindmonitor_mt_tailcall_jit.o
+//
+// For bpf code: clang -target bpf -O2 -Werror -c bindmonitor_mt_tailcall.c -o bindmonitor_mt_tailcall.o
+// this passes the checker
+
+// Whenever this sample program changes, bpf2c_tests will fail unless the
+// expected files in tests\bpf2c_tests\expected are updated. The following
+// script can be used to regenerate the expected files:
+//     generate_expected_bpf2c_output.ps1
+//
+// Usage:
+// .\scripts\generate_expected_bpf2c_output.ps1 <build_output_path>
+// Example:
+// .\scripts\generate_expected_bpf2c_output.ps1 .\x64\Debug\
+
+#include "bpf_helpers.h"
+#include "ebpf_nethooks.h"
+
+// Define a macro for the maximum number of tail call programs
+#define MAX_TAIL_CALL_PROGS 32
```

Comment:
Why isn't the existing define for `MAX_TAIL_CALL_CNT` in `bpf_helpers.h` (which this already includes at line 19) sufficient?  I think you can delete this and just use that.